playbooks/centurion_teams.yaml

```diff
@@ -0,0 +1,244 @@
+- name: Centurion ERP Teams Setup
```

Comment:
# To Do

when adding links directly use the link. i.e. 

- https://github.com/nofusscomputing/ansible_collection_centurion/issues/14

    this format is `- https://github.com/nofusscomputing/ansible_collection_centurion/issues/14`

using this method enables the rendered link to update.